package-configuration/linux-cli/get-cloudify.py

```diff
@@ -263,20 +263,36 @@ def install_pip(self):
             sys.exit('Could not install pip')
 
     def install_pythondev(self):
+        """Installs python-dev and gcc
+
+        This will try to match a command for your distribution.
+        The reason for looping through the names of the distros instead of just
+        checking for equality is that the name of the distro `DISTRO` might not
+        always be a pure distro name. For instance, in CentOS 7.1,
+        platform.linux_distribution() return 'CentOS Linux'. Checking for the
```

Comment:
not entierly correct, you just need to call platform.linux_distribution(full_distribution_name=False) to get the short name. https://docs.python.org/2/library/platform.html#platform.linux_distribution